packages/core/src/common/constants.ts

```diff
@@ -0,0 +1,9 @@
+export const leastSevere = 3;
+export const mostSevere = 1;
+
+// change when sounds are determined
+export const defaultAudioSrc = [
+  'https://audio-alert-sounds.s3.amazonaws.com/Cute-Chime-2.mp3',
+  'https://audio-alert-sounds.s3.amazonaws.com/new-message-3.ogg',
```

Comment:
Why the difference in file type?

Reply:
I just found a few royalty free sound samples online and uploaded it to my own S3 bucket to play with. These sounds aren't defined yet and I will have to review with Mahti, Courtney, and Tracy to finalize what the default sounds will be.
They're both audio file types, OGG is just has better sound quality, a smaller file size, and higher bitrate.